### PR TITLE
Webcam example

### LINKED ISSUE #14: How can we use the webcam?
Can a webcam usage example be provided?
This is a very cool module!
Thanks!

### PR DESCRIPTION
**Pull request by lcristianiim**
Fixes #14 
This feature :truck:
- [x] adding webcam.js
- [x] modify webcam.sh to use webcam.js
- [x] updates README.md with modifications made

### CODE REVIEW

File: test/webcam.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var ImageToAscii = require("../lib/index");
```
Comment (IonicaBizau) [inline test/webcam.js]: Just `../lib` is enough.

**Comment by IonicaBizau** [comment]
...and add yourself as contributor in `package.json`. :smile:

**Comment by lcristianiim** [comment]
@IonicaBizau  check it out now.

**Comment by IonicaBizau** [comment]
:+1:

**Comment by IonicaBizau** [comment]
:ship: `2.0.0` :tada:

**Pull request merged**